cmd/logcounter/log_counter.go

```diff
@@ -28,6 +29,12 @@ import (
 )
 
 func main() {
+	// Do not log to files.
+	if err := flag.Set("logtostderr", "true"); err != nil {
+		fmt.Printf("Failed to set logtostderr=true: %v", err)
```

Comment:
I think this should return unknown error code

Reply:
Done